src/blockchain/milestones.js

```diff
@@ -48,10 +46,10 @@ const milestonesFactory = app => {
             '-> status:',
             status,
           );
-          return;
+          return null;
         }
 
-        await milestones.patch(
+        return milestones.patch(
```

Comment:
Instead of await and return nothing, I returned the promise, to can write assertion for function output in test file